src/main/java/at/hannibal2/skyhanni/utils/GuiRenderUtils.kt

```diff
@@ -314,50 +318,68 @@ object GuiRenderUtils {
         y: Float,
         scaleMultiplier: Double = NeuItems.ITEM_FONT_SIZE,
         rescaleSkulls: Boolean = true,
+        rotationDegrees: Vec3? = null,
     ) {
         val item = checkBlinkItem()
         val isSkull = rescaleSkulls && item.item === Items.skull
 
-        val baseScale = (if (isSkull) 4f / 3f else 1f)
+        val rotX = ((rotationDegrees?.xCoord ?: 0.0) % 360).toFloat()
+        val rotY = ((rotationDegrees?.yCoord ?: 0.0) % 360).toFloat()
+        val rotZ = ((rotationDegrees?.zCoord ?: 0.0) % 360).toFloat()
+
+        val baseScale = if (isSkull) (4f / 3f) else 1f
         val finalScale = (baseScale * scaleMultiplier).toFloat()
 
-        val translateX: Float
-        val translateY: Float
-        if (isSkull) {
-            val skullDiff = ((scaleMultiplier) * 2.5).toFloat()
-            translateX = x - skullDiff
-            translateY = y - skullDiff
-        } else {
-            translateX = x
-            translateY = y
+        val (translateX, translateY) = if (isSkull) {
+            val skullDiff = ((scaleMultiplier) * 2.5f).toFloat()
+            x - skullDiff to y - skullDiff
+        } else x to y
+
+        val halfIconX = 8f
+        val halfIconY = 8f
+        val halfIconZ = 100f
+
+        DrawContextUtils.pushPop {
+            DrawContextUtils.translate(translateX, translateY, -19f)
```

Comment:
```suggestion
        DrawContextUtils.translated(translateX, translateY, -19f) {
```
:3 (its the same thing just in one line)